cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -189,7 +189,8 @@ public class AppLifecycleHttpHandler extends AbstractAppFabricHttpHandler {
   @AuditPolicy(AuditDetail.REQUEST_BODY)
   public BodyConsumer create(HttpRequest request, HttpResponder responder,
       @PathParam("namespace-id") final String namespaceId,
-      @PathParam("app-id") final String appId)
+      @PathParam("app-id") final String appId,
+      @QueryParam("skipMakingLatest") boolean skipMakingLatest)
```

Comment:
This is the public API, we don't want the flag here. Internally, our app service can support adding a new application version without making it the latest, which the SCM logic will use. But end users should not be able to do that.

Reply:
Removed from public API. Added an internal API to be used by scm operations.